_data-prepper/core-apis.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: Core APIs
+nav_order: 2
+---
+
+# Core APIs
+
+All Data Prepper instances expose a server with some control APIs. By default, this server runs on port 4900. Some plugins, especially Source plugins may expose other servers. These will be on different ports and their configurations are independent of the core API. For example, to shut down Data Prepper, you can run:
+
+```
+curl -X POST http://localhost:4900/shutdown
+```
+
+## APIs
+
+The following APIs are available:
+
+| Name | Description |
+| --- | --- | 
+| ```GET /list```<br>```POST /list``` | Returns a list of running pipelines. |
+| ```POST /shutdown``` | Starts a graceful shutdown of Data Prepper. |
+| ```GET /metrics/prometheus```<br>```POST /metrics/prometheus``` | Returns a scrape of Data Prepper metrics in Prometheus text format. This API is available provided `metricsRegistries` parameter in data prepper configuration file `data-prepper-config.yaml` has `Prometheus` as one of the registry.
```

Comment:
The last sentence of this description needs revision for clarity, grammar, and capitalization.

Reply:
@natebower Adjusted wording, but may need to be adjusted again.

Reply:
Made a few edits, but otherwise good.